### PR TITLE
add code to insert position attributes directly

### PR DESCRIPTION
**Pull request by arnej27959**
@geirst please review and merge
@toregge FYI
@bratseth FYI
@havardpe FYI

### CODE REVIEW

File: searchsummary/src/vespa/searchsummary/docsummary/positionsdfw.cpp [hunk 1]
```diff
@@ -113,6 +115,63 @@ PositionsDFW::PositionsDFW(const vespalib::string & attrName) :
...
+        latlong << "E" << degrees_ew;
+    }
+    obj.setString("latlong", vespalib::Memory(latlong.str()));
+}
+
+void
+insertFromAttr(const attribute::IAttributeVector &attribute, uint32_t docid, vespalib::slime::Inserter &target)
+{
+    std::vector<IAttributeVector::largeint_t> pos(16);
+    uint32_t numValues = attribute.get(docid, &pos[0], pos.size());
```
Comment (geirst) [inline searchsummary/src/vespa/searchsummary/docsummary/positionsdfw.cpp]: I suggest using IntegerContent::fill() (searchcommon/src/vespa/searchcommon/attribute/attributecontent.h) instead as that handles resizing and re-trying if necessary.

**Review by geirst — CHANGES_REQUESTED** [review]
Overall looks good, but I suggest simplifying how values are extracted from the attribute vector.

File: searchsummary/src/vespa/searchsummary/docsummary/positionsdfw.cpp (hunk 1 above)
Comment (arnej27959) [reply searchsummary/src/vespa/searchsummary/docsummary/positionsdfw.cpp]: done.

**Comment by arnej27959** [comment]
@geirst please take another look

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**